adapter-http/src/main/resources/activities/baselines/http-rest-keyvalue.yaml

```diff
@@ -21,7 +21,7 @@ bindings:
   weighted_hosts: WeightedStrings('<<restapi_host:stargate>>')
 
   request_id: ToHashedUUID(); ToString();
-  request_token: ToString(); TextOfFile("TEMPLATE(stargate_tokenfile,data/stargate_token.txt)")
+  request_token: ToString(); TextOfFile(TEMPLATE(stargate_tokenfile,'data/stargate_token.txt'));
```

Comment:
The quotes on this are part of the value, but should probably be outside the TEMPLATE macro.

Reply:
Added.